app/uk/gov/hmrc/helptosavefrontend/views/core/sidebar.scala.html

```diff
@@ -0,0 +1,23 @@
+@*
```

Comment:
`sidebar` sounds too general for something that is only used by the `not_eligible` page. `not_eligible_sidebar` would be more accurate or you could even do something like this in the HTML inline in the `not_eligible` file:

```scala
@(ineligibilityType: IneligibilityType)(implicit htsContext: HtsContext, request: Request[_], messages: Messages)

@y = @{"""<section class="section subsection section--blue-top">
             <h2 class="heading-medium">@Messages("hts.core.not-eligible.sidebar.h2")</h2>
             <p class="font-xsmall">
             <a href="https://www.gov.uk/working-tax-credit">@Messages("hts.core.not-eligible.sidebar.p")</a></p></section>"""}

@uk.gov.hmrc.helptosavefrontend.views.html.main_template(
    title = Messages("hts.core.not-eligible.page-title"),
    sidebarLinks = Some(Html(y)),
    bodyClasses = None) {
    ...
```